gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/dsl/ExperimentalUiKitApplication.kt

```diff
@@ -5,25 +5,13 @@
 
 package org.jetbrains.compose.experimental.dsl
 
-import org.gradle.api.Action
 import org.gradle.api.model.ObjectFactory
-import org.jetbrains.compose.internal.requiredDslProperty
 import javax.inject.Inject
 
 @Suppress("unused")
 abstract class ExperimentalUiKitApplication @Inject constructor(
```

Comment:
Do we still need `ExperimentalUiKitApplication` and `ExperimentalUiKitExtension`?

It seems there are not used now.

Reply:
Yes, we can remove it too

Reply:
Done